src/App.jsx

```diff
@@ -5,31 +5,34 @@ import './App.css'
 
 function App() {
   const [count, setCount] = useState(0)
```

Comment:
```suggestion
```
Den state können wir entfernen (brauchen wir nicht mehr)

Reply:
den habe ich bewusst drin gelassen, den nutze ich in step-3